tower-http/Cargo.toml

```diff
@@ -21,7 +21,7 @@ tower-layer = "0.3"
 tower-service = "0.3"
 
 # optional dependencies
-# will be filled out in other PRs
+tracing = { version = "0.1", optional = true }
```

Comment:
nit: we should probably set 
```suggestion
tracing = { version = "0.1", optional = true, default-features = false }
```
so that users who are depending on tracing with its default features disabled can still use this without having them all be turned on — we don't use anything that's feature-flagged in `tracing`.